### ISSUE TITLE
FEATURE REQUEST: Add Ability to Compress into Cabinet File

### ISSUE THREAD

**Issue by Lesita** [Leafuke/MineBackup#1]
Do you have any plans to add the abilitiy to compress into Microsoft Cabinet files in order to enable native support instead of using 7-Zip technology?

**Comment by Leafuke** [comment]
Of course! To be honest this is the first time I realized I could use makecab to compress files, which saves the trouble of downloading 7-Zip. I will try to add this feature in the next version. For example, when it detects that 7-Zip is not installed on the computer it will use Cabinet without the need for an additional download.

**Comment by Leafuke** [comment]
For technical reasons, support for makecab has not been added in [v1.2.1-beta](https://github.com/Leafuke/MineBackup/releases/tag/v1.2.1-beta), but there is currently a "portable version" that allows us to use [MineBackup](https://github.com/Leafuke/MineBackup) without having to install 7-zip.
I think that's a workaround as well. If you, or others, still insist on adding support for makecab, I'll continue to try it.😁

**Comment by Leafuke** [comment]
In the latest version, you no longer need to download 7-Zip to get high-compression backups! It has been integrated into the program.

**Comment by MorningMC** [comment]
why not add a option to choose the compression method?

**Comment by Leafuke** [comment]
> why not add a option to choose the compression methods and arguments?

Thank you for your suggestion regarding the addition of more compression methods and arguments.

After careful consideration, I've decided not to implement this feature for the time being. Our primary reasons are:

1. Maintainability & Stability: Introducing additional compression formats (like .cab) or a wide array of arguments would significantly increase the complexity of the codebase. This complicates testing, raises the potential for edge-case bugs, and could undermine the current stability of the backup and restore process, which is our top priority.

2. Limited Benefit for the Use Case: For Minecraft world data, which is primarily comprised of already compressed binary files (like .mca), the compression ratio gains from alternative formats are often minimal. The current use of robust, widely-supported formats like the one provided by `7z` or `zip` offers an excellent balance of size and speed.

While I won't be adding this as a built-in feature now, I value community input. If there's a strong and specific use case that isn't met by the current implementation, please feel free to provide more details on this issue.
